### DISCUSSION TITLE
What's the best browser experience?

### DISCUSSION THREAD

**Discussion by garrett** [uxbox/uxbox#550] [Q&A] [answered]
Which browser works best with Penpot?

- Firefox
- Chrome (or Chromium/Blink based browsers)
- WebKit (Safari / Epiphany)

Is there one that has more bugs than the others, or are they all supposed to work just as well?

Is Penpot routinely tested with each?

**Comment by elhombretecla** [comment, marked as answer]
Hi Garret! Thank you for your question.

We are doing as much browser testing and support as possible. If I have to give you a list based on usage preference I would say:

- Chrome (or Chromium/Blink based browsers/Edge)
- Firefox
- WebKit (Safari / Epiphany)

**Reply by garrett** [reply]
Awesome, thanks! I'm happy that you're consdering Firefox and WebKit browsers too. (It was working well in Firefox when I tried it, so I assumed you either have been testing it or got very lucky to have it work so well. :grin:)

**Reply by elhombretecla** [reply]
Hahahaha yes!!! We are so lucky to have users like you! But the browser support and testing is hell xD

However, we try to cover as much browsers as possible to give you the best experience :)
